examples/cpp/power/power.h

```diff
@@ -7,7 +7,7 @@ extern "C"
   
   #include <stdint.h>
   #include <stdbool.h>
-  int32_t power_power_of(int32_t base, int32_t exp);
+  int32_t power_power_of(int32_t b, int32_t exp);
```

Comment:
?

Reply:
Lol... in the python bindings, wit-bindgen uses internal variables with the names `base`, `i`, and one or two other common argument names, so there is a naming collision when we run writ.  I've filed a bug for this.  In the meantime, I wanted to avoid talking about it in the tutorial so I just changed the name of the argument we use in our example for now.

Reply:
Ah ok so for this reason we need to commit things (without my diff). Maybe next I'll put in a few fixes for wit-bindgen. I filed an issue last week for printing DO NOT EDIT docstrings. Once we have those two fixes in wit-bindgen, we should remove the generated code.

Reply:
I came up with a slightly different solution by fixing the generated file as part of our make script. Now that the Makefiles produce something that doesn't need manual changes, I opted to remove them from git.